libs/ui/src/lib/containers/CardHeader/CarHeader.style.ts

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+// eslint-disable-next-line import/no-unresolved
+import { pxToRem } from '@guiabolsobr/utils';
+import { themeProps } from '@guiabolsobr/ui';
+import { Text } from '../../display/Text';
+
+import { CustomThemeProps } from '../../../typings/CustomThemeProps';
+
+type HeaderProps = {
+  hasSuptitle: boolean;
```

Comment:
Fiquei na dúvida se é realmente `hasSuptitle` ou se deveria ser `hasSu_B_title`

```suggestion
  hasSubtitle: boolean;
```

Reply:
mano, como é em cima, acho que o correto mesmo é su**p**title